deployment/Dockerfile.debian.amd64

```diff
@@ -14,7 +14,7 @@ RUN apt-get update -yqq \
   && apt-get install -yqq --no-install-recommends \
     debhelper gnupg devscripts build-essential mmv \
     libcurl4-openssl-dev libfontconfig1-dev libfreetype6-dev libssl-dev \
-    libssl1.1 liblttng-ust0
+    libssl3 liblttng-ust1
```

Comment:
Please don't change anything about Debian packaging. This has a lot more implications than just changing these values imply.